docker/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "3"
+services:
+  conpot:
+    build: ./
+    ports:
+     - "80:8800" #SCADA UI, http
+     - "102:10201" #S7Comm
+     - "502:5020" #Modbus
+     - "161:16100/udp" #SNMP
+     - "47808:47808" #Bacnet
+     - "623:6230" #IPMI
+     - "21:2121" #FTP
+     - "69:6969/udp" #TFTP
```

Comment:
I think you forgot about EN/IP protocol (`44818:44818`) :slightly_smiling_face: 

We support several other protocols as well. These include : Kamstrup meter/management protocol, IEC104 and Guardian Ast via different templates. I don't expect you to include these in this PR, but it would be awesome if you could :wink: